Sponsorship page update (#1453)
jhipster/jhipster.github.io: docs/contributing/sponsors.mdx

title: Sponsors
slug: /sponsors/
last_update:
  date: 2025-02-01T00:00:00-00:00
---

JHipster uses [Open Collective](https://opencollective.com/generator-jhipster) to gather money. This money is used to cover project expenses (like running this website) in a transparent way ([See the Open Collective documentation](https://opencollective.com/how-it-works)), and your donation will help the project live and grow successfully.

Please go to our [Open Collective page](https://opencollective.com/generator-jhipster) to become a backer or a sponsor.

## Disclaimer

This is a donation. No goods or services are expected in return. Any requests for refunds for those purposes will be rejected.

We are a French non-profit organization, as descibed in our [status](/association/), and we're not interested in business partnerships.

## Benefits of being a backer

Backers donate $2 per month to the project.

> We use the OpenCollective APIs to automatically display backers on our main page:

Reviewer: maybe :

```suggestion
We use the Open Collective APIs to automatically display backers on our main page:
```

if it's ok, your should do the same for other text like this

- On the front page of [https://www.jhipster.tech](https://www.jhipster.tech) in the "backers" section (about 110,000 views/month).
- On the GitHub main project page in the "backers" section (about 15,000 views/month).

## Benefits of being a bronze sponsor

Bronze sponsors donate $100 per month to the project.

We use the OpenCollective APIs to automatically display bronze sponsors on our main page:

- On the front page of [https://www.jhipster.tech](https://www.jhipster.tech) in the "sponsors" section (about 110,000 views/month).
- On the GitHub main project page in the "sponsors" section (about 15,000 views/month).

## Benefits of being a silver sponsor

There is a limit to 10 silver sponsors.

> Bronze sponsors donate $500 per month to the project.

Reviewer 2: ```suggestion
Silver sponsors donate $500 per month to the project.
```

Looks like a mistake here.

title: Sponsors
slug: /sponsors/
last_update:
  date: 2025-02-01T00:00:00-00:00
---

JHipster uses [Open Collective](https://opencollective.com/generator-jhipster) to gather money. This money is used to cover project expenses (like running this website) in a transparent way ([See the Open Collective documentation](https://opencollective.com/how-it-works)), and your donation will help the project live and grow successfully.

Please go to our [Open Collective page](https://opencollective.com/generator-jhipster) to become a backer or a sponsor.

## Disclaimer

This is a donation. No goods or services are expected in return. Any requests for refunds for those purposes will be rejected.

> We are a French non-profit organization, as descibed in our [status](/association/), and we're not interested in business partnerships.

Reviewer 3: ```suggestion
We are a French non-profit organization, as described in our [status](/association/), and we're not interested in business partnerships.
```
